### ISSUE TITLE
Robopods for Admob + iOS 12, freezing

### ISSUE THREAD

**Issue by aberkowski** [MobiDevelop/robovm-robopods#62]
My game suffer from random freezes when using iOS 12. I know that there was a bug with robovm and iOS 12 but I use the most recent version of robovm (2.3.5) and robopods-google-mobile-ads-ios (2.2.2).
I've created sample app with banner ad (I use admob test banner ad unit id) and some code to run garbage collector. If I disable 'causeGC()' method or do not initialize ads everything works fine. To run a sample project you have to download Google Mobile Ads SDK for iOS (https://developers.google.com/admob/ios/download) and place it in folder 'ios/libs/'. Link to sample app: https://github.com/Berobasket/TestLibgdxGame . It takes from 2 to 70 seconds to freeze on my IPhone SE.

**Comment by obigu** [comment]
Just to confirm, the freezes reproducing when using the latest version of Google Mobile Ads which currently is 7.34.0 right?

I use Robopods 2.2.3-SNAPSHOT (they're exactly the same as 2.2.2) and MobiVM 2.3.5 on several live games with Google Mobile Ads SDK 7.30.0 and have only integrated banner ads but haven't experienced freezes.

What I've experienced is lag when banners are requested which is specially noticeable in case the banner is shown permanantly in game (instead of showing/hiding in natural pauses).

Having a quick look at the project (haven't run it) I see you request Landscape banners but your project is Portrait. Doubt is related but who knows.

**Comment by aberkowski** [comment]
Yes, Google Mobile Ads 7.34.0. I've changed banner ad to Portrait (thanks for spot) and it freeze only 1 time for 3 runs. But when I change project to Landscape and banner ad to Landscape it freezed 3 times for 3 runs.. Twice after 3 seconds and once after 81 seconds. So, maybe it is something related to landscape banner ad? After freeze I got info: "void SendDelegateMessage(NSInvocation *): delegate (webview:decidePolicyForNavigationAction:request:frame:decisionListener:) failed to return after waiting 10.000000 seconds. main run loop mode: kCFRunLoopDefaultMode"
